### PR TITLE
drone: Resign pipeline for drone.teleport.dev

### PR DESCRIPTION
**Pull request by webvictim**
Runs `make dronegen` to resign the Drone pipeline for https://drone.teleport.dev

This also adds `privileged: true` to all uses of `docker:dind` to provide the necessary privileged environment for running Docker-in-Docker. This wasn't needed on the previous Drone instance because `docker:dind` was a "whitelisted" image at the Drone level (and therefore always ran in privileged mode) but in the interests of security and clarity, @wadells and I decided to disable this behaviour on the new Drone instance and be explicit when privileged mode is being used.

Backports/versions required:
- `branch/v6.2`
- `branch/v6`
- `branch/5.0`
- `branch/4.4`

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -186,6 +186,7 @@ steps:
 services:
 - name: Start Docker
   image: docker:dind
+  privileged: true
```
Comment (r0mant) [inline .drone.yml:189]: Sorry I don't have a lot of context on Drone migration but I do have a couple of questions looking at the changeset:

* Why is the privileged flag now required for all steps?
* How was it working before without privileged containers?
* Do all steps need privileged containers (seems like it got added to all)?

Comment (webvictim) [reply .drone.yml:189]: Yeah, I just realised this wasn't helpful. I've added some clarity in the top PR comment.

**Review by r0mant — APPROVED** [review]

File: .drone.yml (hunk 1 above)
Comment (r0mant) [reply .drone.yml:189]: Thank you @webvictim, it makes sense.

**Review by wadells — APPROVED** [review]

**Review by quinqu — APPROVED** [review]

**Review by russjones — APPROVED** [review]
Bot.

**Pull request merged**